src/AppInstallerCLICore/ChannelStreams.h

```diff
@@ -106,28 +114,10 @@ namespace AppInstaller::CLI::Execution
         // Applies the format for the stream.
         void ApplyFormat();
 
-        BaseStream m_out;
+        BaseStream& m_out;
+        bool m_enabled;
+        bool m_VTEnabled;
         size_t m_applyFormatAtOne = 1;
         VirtualTerminal::ConstructedSequence m_format;
     };
-
-    // Does not allow VT at all.
-    struct NoVTStream
-    {
-        NoVTStream(std::ostream& out, bool enabled);
-
-        template <typename T>
-        NoVTStream& operator<<(const T& t)
-        {
-            m_out << t;
-            return *this;
-        }
-
-        NoVTStream& operator<<(std::ostream& (__cdecl* f)(std::ostream&));
-        NoVTStream& operator<<(const VirtualTerminal::Sequence& sequence) = delete;

```

Comment:
The other thing that the `NoVTStream` did was make it a compile error to attempt to output a VT sequence.  Not the end of the world to not have it, but I do like the very early notification that it won't work.